Fix documentation build errors with Texinfo 5 and Perl 5.18

wget.pod around line 2105: Expected text after =item, not a number
wget.pod around line 2110: Expected text after =item, not a number
wget.pod around line 2116: Expected text after =item, not a number
wget.pod around line 2121: Expected text after =item, not a number
wget.pod around line 2126: Expected text after =item, not a number
wget.pod around line 2131: Expected text after =item, not a number
wget.pod around line 2136: Expected text after =item, not a number
wget.pod around line 2141: Expected text after =item, not a number
wget.texi:879: @itemx must follow @item
wget.texi:980: @itemx must follow @item
wget.texi:3097: @itemx must follow @item
wget.texi:3608: warning: @itemx should not begin @table

Upstream-Status: Backport
Signed-off-by: Jonathan Liu <net147@gmail.com>

diff --git a/doc/texi2pod.pl b/doc/texi2pod.pl
index 86c4b18..9db6de1 100755
--- a/doc/texi2pod.pl
+++ b/doc/texi2pod.pl
@@ -291,7 +291,7 @@ while(<$inf>) {
 	if (defined $1) {
             my $thing = $1;
             if ($ic =~ /\@asis/) {
-                $_ = "\n=item $thing\n";
+                $_ = "\n=item C<$thing>\n";
             } else {
                 # Entity escapes prevent munging by the <> processing below.
                 $_ = "\n=item $ic\&LT;$thing\&GT;\n";
diff --git a/doc/wget.texi b/doc/wget.texi
index 7efdc72..2b045fd 100644
--- a/doc/wget.texi
+++ b/doc/wget.texi
@@ -876,7 +876,7 @@ recommendation to block many unrelated users from a web site due to the
 actions of one.
 
 @cindex proxy
-@itemx --no-proxy
+@item --no-proxy
 Don't use proxies, even if the appropriate @code{*_proxy} environment
 variable is defined.
 
@@ -977,7 +977,7 @@ are outside the range of @sc{ascii} characters (that is, greater than
 whose encoding does not match the one used locally.
 
 @cindex IPv6
-@itemx -4
+@item -4
 @itemx --inet4-only
 @itemx -6
 @itemx --inet6-only
@@ -3094,7 +3094,7 @@ display properly---the same as @samp{-p}.
 Change setting of passive @sc{ftp}, equivalent to the
 @samp{--passive-ftp} option.
 
-@itemx password = @var{string}
+@item password = @var{string}
 Specify password @var{string} for both @sc{ftp} and @sc{http} file retrieval. 
 This command can be overridden using the @samp{ftp_password} and 
 @samp{http_password} command for @sc{ftp} and @sc{http} respectively.
@@ -3605,7 +3605,7 @@ In addition to the environment variables, proxy location and settings
 may be specified from within Wget itself.
 
 @table @samp
-@itemx --no-proxy
+@item --no-proxy
 @itemx proxy = on/off
 This option and the corresponding command may be used to suppress the
 use of proxy, even if the appropriate environment variables are set.
